core/util/email.py

```diff
@@ -0,0 +1,23 @@
+import os
+
+from redmail import EmailSender
+
+
+class EmailManager:
+    MAIL_SERVER = os.environ.get("SIMPLIFIED_MAIL_SERVER")
+    MAIL_PORT = int(os.environ.get("SIMPLIFIED_MAIL_PORT", "25"))
+    MAIL_USERNAME = os.environ.get("SIMPLIFIED_MAIL_USERNAME")
+    MAIL_PASSWORD = os.environ.get("SIMPLIFIED_MAIL_PASSWORD")
```

Comment:
The sender could live as an environment variable here.

Reply:
Maybe call it `SIMPLIFIED_MAIL_SENDER`. That way @io7m can add it to https://github.com/ThePalaceProject/hosting-playbook/pull/329 so we can get this deployed out.

Reply:
Sure, makes sense 👍